services/catalog/requirements/_test.in

```diff
@@ -15,6 +15,7 @@ jsonschema
 # testing
 pytest
 pytest-aiohttp
+pytest-asyncio
```

Comment:
this should not be necessary. look at catalog _test.txt.
pytest-asyncio is a dependency of pytest-aiohttp. Please remove it.

Reply:
done